[2024-10-17T22:09:42.366+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T22:09:42.399+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.load_data_into_Redshif scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T22:09:42.419+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.load_data_into_Redshif scheduled__2024-10-16T00:00:00+00:00 [queued]>
[2024-10-17T22:09:42.423+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-17T22:09:42.499+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_data_into_Redshif> on 2024-10-16 00:00:00+00:00
[2024-10-17T22:09:42.513+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=190) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-17T22:09:42.516+0000] {standard_task_runner.py:72} INFO - Started process 194 to run task
[2024-10-17T22:09:42.518+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'load_data_into_Redshif', 'scheduled__2024-10-16T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmpff63ozwu']
[2024-10-17T22:09:42.524+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask load_data_into_Redshif
[2024-10-17T22:09:42.626+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.load_data_into_Redshif scheduled__2024-10-16T00:00:00+00:00 [running]> on host d95e36bcd28a
[2024-10-17T22:09:42.770+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='load_data_into_Redshif' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-16T00:00:00+00:00'
[2024-10-17T22:09:42.772+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T22:09:45.671+0000] {logging_mixin.py:190} INFO - Connection to Redshift successful!
[2024-10-17T22:09:45.983+0000] {logging_mixin.py:190} INFO - update from Redshift: 2024-10-17
[2024-10-17T22:09:46.289+0000] {logging_mixin.py:190} INFO - Error reading file: Bad magic number for file header
